### PR TITLE
I811 fix chart arrows

### PR DESCRIPTION
**Pull request by Serhiy91**

### CODE REVIEW

File: app/imports/ui/react/components/Timeline/TimelineHorizontalLine.js [hunk 1]
```diff
@@ -15,32 +15,39 @@ const enhance = withProps(({
...
         y: index,
-        x: start,
+        x: start || startDate,
         title: startDate,
         fill: color,
         isStart: true,
         label: 'Start Date',
-        symbol: isStartWithinScale ? 'circle' : 'arrowLeft',
-        strokeWidth: isStartWithinScale ? 10 : 4,
+        titleAnchor: isLineOutOfScaleRight ? 'end' : 'start',
```
Comment (Dartv) [inline app/imports/ui/react/components/Timeline/TimelineHorizontalLine.js]: shouldn't it be `textAnchor`?

File: app/imports/ui/react/components/Timeline/TimelinePoint.js [hunk 2]
```diff
@@ -50,7 +52,7 @@ const TimelinePoint = ({
   togglePopover,
   renderPopover,
   isLabel,
-  textAnchor,
+  titleAnchor,
```
Comment (Dartv) [inline app/imports/ui/react/components/Timeline/TimelinePoint.js]: ?

File: app/imports/ui/react/components/Timeline/TimelineHorizontalLine.js (hunk 1 above)
Comment (Serhiy91) [reply app/imports/ui/react/components/Timeline/TimelineHorizontalLine.js]: @Dartv 
textAnchor is point prop
titleAnchor is point label prop
I need to change point label Anchor (titleAnchor)

**Comment by Serhiy91** [comment]
@Dartv fixed

**Pull request merged**